### ISSUE TITLE
False positive when using an interpolated string in a map

### ISSUE THREAD

**Issue by flaviogrossi** [rrrene/credo#542]
### Environment

* Credo version: 0.9.2
* Erlang/Elixir version (`elixir -v`): 1.6.5
* Operating system: Linux

### What were you trying to do?
Checking the following module:

```elixir
defmodule Test do
  def hello do
    x = "hello"
    %{"some-atom-with-quotes": "#{hello} world"}
  end
end
```

### Expected outcome

No warnings

### Actual outcome

The following warning:
```  Consistency                                                                   
┃ 
┃ [C] ↗ There is no whitespace around parentheses/brackets most of the time, but 
┃       here there is.
┃       lib/test.ex:4 #(Test.hello)
```
Using as a key an atom with no quotes, or removing the string interpolation in the value does not produce the warning.

**Comment by rrrene** [comment]
@flaviogrossi @jestingrabbit Could you test using Elixir 1.7 and Credo 0.10.0 - this should be fixed now :+1:

**Comment by flaviogrossi** [comment]
> @flaviogrossi @jestingrabbit Could you test using Elixir 1.7 and Credo 0.10.0 - this should be fixed now

I can confirm it works fine now. Thanks

**Comment by rrrene** [comment]
Great!
